src/cobalt.rs

```diff
@@ -33,6 +33,13 @@ fn compare_paths(a: &Path, b: &Path) -> bool {
     }
 }
 
+fn starts_with_path(complete_path: &Path, post_path: &Path) -> bool {
```

Comment:
this function signature looks slightly confusing to me: argument name `post_path` sounds like this is a path to a single post (`posts_path` makes more sense here).
it also may make sense to avoid `post_path`/`posts_path` completely - the function can be reused then. what about `starts_with_path(this: &Path, path: &Path)`?

could you please also add a docstring to improve readability?